jenkins_pipelines/environments/common/pipeline-build-validation.groovy

```diff
@@ -261,6 +261,7 @@ def clientTestingStages() {
 
     //Get minion list from terraform state list command
     def minionList = getMinionList()
+    def mu_sync_status = minionList.MUSyncStatus
```

Comment:
I'm fine with this PR.
But I think it would be good to make some adjust with the names of some variables.

For instance:
```
minionList.nodeList.each { minion ->
```
- If we are iterating from "nodeList" we should be having as item "node".
- If we already have "minionList", we would guess by its name that we can iterate from it the list of minions, but instead we must use an internal attribute for that purpose. Should we name this variable "minionList" differently, I propose to call it "nodesHandler" and so "getNodesHandler" by the nature of its content.

Reply:
I update this naming to the pipeline